index.html

```diff
@@ -87,6 +91,8 @@
             <br>
         </div>
         <br>
+        <div class="plyr"><audio></audio></div>
```

Comment:
It doesn't work locally for me though.  
I'm pretty sure Plyr checks what's inside the div we're running Plyr on while initializing. It checks for `<div>`, `<audio>`, and `<video>` (used for YT and Vimeo, audio files, and video files respectively). When calling `.source` Plyr takes care of pushing new html elements but but before that, with empty div, it doesn't want to initialize properly for me.